### PR TITLE
5525 custom email templates

### PR DESCRIPTION
**Pull request by tavib47**

### CODE REVIEW

File: docroot/modules/iucn/iucn_notifications/iucn_notifications.module [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  $message['body'][] = $params['message'];
+  $message['headers']['Content-Type'] = 'text/html; charset=UTF-8; format=flowed; delsp=yes';
+}
+
+/**
+ * Implements hook_node_presave().
+ */
+function iucn_notifications_node_presave(\Drupal\Core\Entity\EntityInterface $node) {
+  if ($node->bundle() == 'site_assessment') {
+    $tokens = [];
```
Comment (tavib47) [inline docroot/modules/iucn/iucn_notifications/iucn_notifications.module]: `$tokens` variable doesn't get filled. Is it ok? If yes, change the lines below `sendNotification(NotificationService::$WORKFLOW_SETUP_NEW_ASSESSMENT, $to, ['tokens' => $tokens]);` to `sendNotification(NotificationService::$WORKFLOW_SETUP_NEW_ASSESSMENT, $to);`

Comment (tavib47) [reply docroot/modules/iucn/iucn_notifications/iucn_notifications.module]: @scovitin-vsevolod

Comment (tavib47) [reply docroot/modules/iucn/iucn_notifications/iucn_notifications.module]: The same happens in `iucn_notifications_entity_update` function

File: docroot/modules/iucn/iucn_notifications/iucn_notifications.module [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    }
+  }
+}
+
+/**
+ * Implements hook_mail_alter
+ */
+function iucn_notifications_mail_alter(&$message) {
+  // Overrides default user module notifications
+  drupal_set_message($message['module']  . ' iucn_notifications_mail_alter => ' . $message['key']);
```
Comment (tavib47) [inline docroot/modules/iucn/iucn_notifications/iucn_notifications.module]: @scovitin-vsevolod What is this? :)

Comment (scovitin-vsevolod) [reply docroot/modules/iucn/iucn_notifications/iucn_notifications.module]: removed and commited

File: docroot/modules/iucn/iucn_notifications/iucn_notifications.module (hunk 1 above)
Comment (scovitin-vsevolod) [reply docroot/modules/iucn/iucn_notifications/iucn_notifications.module]: @tavib47 
Removed empty tokens

**Pull request merged**